<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMindMapToolPanel</class>
 <widget class="QWidget" name="CMindMapToolPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMindMapToolPanel</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_expandedView" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QWidget" name="m_title" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnCollapse">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/global-collapse.png</normaloff>:/res/icons/global-collapse.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_btnSetting">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>13</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Tool Panel Settings</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../res/mac/res.qrc">
             <normaloff>:/res/icons/global-settings-mini.png</normaloff>:/res/icons/global-settings-mini.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="m_tools" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_collapsedView" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="m_quickTools" native="true"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnExpand">
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../res/mac/res.qrc">
          <normaloff>:/res/icons/global-expand.png</normaloff>:/res/icons/global-expand.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
